src/DataTransferObject.php

```diff
@@ -29,25 +29,43 @@ public function __construct(array $parameters = [])
     {
         $class = new ReflectionClass(static::class);
```

Comment:
IMHO this is unnecessary here.

The DTOCache lookup key is `static::class`, so you only need to on-demand create the reflection class in the cache resolver.

This `$class` is also used further down when throwing `unknownProperties`  error, but I believe you can simply use `static::class` for the name there as well.

Reply:
Good point, that's fixed now!